e2e/bitrise.yml

```diff
@@ -0,0 +1,193 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+
+  test_nested_module:
+    title: Test simple Android project with nested module
+    envs:
+    - SAMPLE_APP_URL: https://github.com/bitrise-io/android-multiple-test-results-sample.git
+    - BRANCH: maintenance
+    - EXPECTED_APK: mylibrary-debug-androidTest.apk
+    before_run:
+    - _setup
+    after_run:
+    - _check_apk
+    steps:
+    - install-missing-android-tools:
```

Comment:
NOTE: We agreed to use a common '_run' or similar workflow for consistency (do we have a written note on this @mateherber ?) (for example https://github.com/bitrise-steplib/steps-git-clone/blob/d10e4259c5d6dbc92b6f816a7af99247d96420cc/e2e/bitrise.yml).